packages/typespec-ts/src/transform/transform.ts

```diff
@@ -84,12 +85,13 @@ export async function transformRLCModel(
     options,
     schemas,
     responses,
-    importSet,
     apiVersionInfo,
     parameters,
     helperDetails,
     urlInfo,
-    telemetryOptions
+    telemetryOptions,
+    innerImports: importSet,
+    thirdPartyImports: build3ndPartyImports(options.branded)
```

Comment:
If I understand this correctly, inner imports track the imports within the same library while thirdPartyImports tracks the imports of core/runtime libraries right?

nit: can we make this a single property imports: { internalImports, externalImports };

Reply:
update to  { internalImports, runtimeImports };